home/filter.go

```diff
@@ -472,7 +472,7 @@ func (f *Filtering) parseFilterContents(file io.Reader) (int, uint32, string) {
 			continue
 		}
 
-		if line[0] == '!' {
+		if line[0] == '!' || line[0] == '#' {
```

Comment:
Unfortunately it won't work, because of `f.filterTitleRegexp = regexp.MustCompile(`^! Title: +(.*)$`)`
This regex expects only `!` at the beginning.

OK, so let's just make it simple, shall we?  Just add your check below like this:

```
if line[0] == '!' {
...
else if line[0] == '#'  {
// skip comment
} else if line[0] == '#'  {
			rulesCount++
		}
```

Reply:
My apologies, I do hope my latest change is what you're looking for.